contribute/style-guides/redux.md

```diff
@@ -117,17 +117,48 @@ Type '{ nonExistingProperty: string; data: string[]; }' is not assignable to typ
   Object literal may only specify known properties, and 'nonExistingProperty' does not exist in type 'ExampleReducerState'. [2322]
 ```
 
-## New test functionality
+## Test functionality
 
 ### reducerTester
 
 Fluent API that simplifies the testing of reducers
 
-#### Example
+#### Simple Example
 
 ```typescript
 reducerTester()
   .givenReducer(someReducer, initialState)
   .whenActionIsDispatched(someAction('reducer tests'))
   .thenStateShouldEqual({ ...initialState, data: 'reducer tests' });
 ```
+
+#### Complex Example
```

Comment:
```suggestion
#### Complex example
```

[Write headings in sentence case, not title case](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings)